Swift/Tests/DatabaseEncryptionTest.swift

```diff
@@ -38,6 +38,7 @@ class DatabaseEncryptionTest: CBLTestCase {
         return try Database(name: "seekrit", config: config)
     }
     
+    #if COUCHBASE_ENTERPRISE
```

Comment:
Why we don't need this previously? The openSeekrit() above also use the enterprise only API. Maybe this file is only in the EE target? Can you double check?